MACILE-guide.md

```diff
@@ -0,0 +1,42 @@
+# MACILE's guide
+
+### Logo
+
+![logo-macile](images/logo-macile.jpg)
+
+### Tipografía
+
+1. `Trajan Pro Bold`
+
+2. `Trajan Pro Regular` 
+
+### Colores
+
+1. **Azul**
+
+![azul](images/azul.png)
+
+- Hex `1a1451`
+- RGB `26, 20, 81`
+
+2. **Dorado**
```

Comment:
It would be good to indicate what these colors are used for! What's the primary color, what's the accent, etc. Maybe we don't know these yet, so that's okay! We can tackle that later even.

Reply:
I opened another issue. We can tackle this later #12